SpiffWorkflow/task.py

```diff
@@ -629,10 +633,18 @@ def run(self):
         })
         metrics.debug('', extra=extra)
         if retval is None:
-            self._set_state(TaskState.WAITING)
+            # This state is intended to indicate a task that is not finished, but can continue
```

Comment:
the first part of this comment might have to do with when it was setting the state to WAITING, and might now be irrelevant? note that i fixed a couple typos in this comment block in a commit on this branch, so please git pull.